pinc/job_log.inc

```diff
@@ -1,20 +1,17 @@
 <?php
 
-function insert_job_log_entry($filename, $event, $comments, $tracetime = null)
+function insert_job_log_entry($filename, $event, $comments, $succeeded = null)
```

Comment:
There are no consumers of this function in here or noncvs that pass in a tracetime, so this seems safe to change.